.nsprc

```diff
@@ -1,5 +1,7 @@
 {
   "exceptions": [
+    "https://nodesecurity.io/advisories/55",
+
```

Comment:
Turns out comments _are_ possible, https://github.com/dominictarr/rc#formatted-as-json
PR updated. Circle-CI is doing its magic as we speak.